en/resources/pacman_start

```diff
@@ -1,20 +1,20 @@
 #!/bin/sh
 
 DIRECTORIES=(
-    /home/pi/Pictures
-    /home/pi/Desktop
-    /home/pi/Templates
-    /home/pi/python_games
-    /home/pi/Documents
-    /home/pi/Documents
-    /home/pi/Music)
+    /home/$USER/Pictures
+    /home/$USER/Desktop
+    /home/$USER/Templates
+    /home/$USER/python_games
+    /home/$USER/Documents
+    /home/$USER/Documents
```

Comment:
Only a minor point, but there's no need to have `/home/$USER/Documents` in here twice :wink: 

And I believe `$HOME` is usually the same as `/home/$USER` ?